internal/pkg/cli/pipeline_show.go

```diff
@@ -199,7 +195,7 @@ func (o *showPipelineOpts) getPipelineNameFromManifest() (string, error) {
 
 // Execute shows details about the pipeline.
 func (o *showPipelineOpts) Execute() error {
-	if o.pipelineName == "" {
+	if o.name == "" {
```

Comment:
Is it true that `o.name` is either passed via the flag or prompted for. If so this seems like a dead if block.